graph/client.py

```diff
@@ -130,12 +152,19 @@ def parse_arguments(self, args):
             action="store_true",
             default=False,
             help="It does not start the web browser.")
+        parser.add_argument(
+            '--tree',
+            action="store_true",
+            default=False,
+            help=("Switch graph_html_interpreter to tree_html_interpreter."
```

Comment:
I think `graph_html_interpreter` and `tree_html_interpreter` are internal details which aren't interesting for the users. Instead, I suggest something like: "Render the graph in a form of directory tree."